qa/L0_client_memory_growth/test.sh

```diff
@@ -96,7 +96,9 @@ for PROTOCOL in http grpc; do
             EXTRA_ARGS="-r ${REPETITION_PY} -i ${PROTOCOL}"
         fi
 
+        SECONDS=0
```

Comment:
Where is SECONDS used?

Reply:
SECONDS is a builtin variable in Bash that tracks the number of seconds that have passed since the shell was started, so I set SECONDS to 0 before starting the following events and calculate the time afterwards.

Reply:
It is an internal bash variable. @krishung5  is setting it to 0 so that $SECONDS captures the test runtime.

Reply:
Thanks for clarifying that. It is a pretty neat feature :)